### PR TITLE
DW-1147-redirect-submit-button-agencies-form

### PR DESCRIPTION
**Pull request by silsanchez8**
## Modifications :  https://makingsense.atlassian.net/browse/DW-1147
* Fix(AgenciesForm): Redirect button to home

image

### CODE REVIEW

File: src/i18n/en.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const urlSiteContact = `${urlSite}/en/contact/`;
 const urlControlPanelMain = `${urlControlPanel}/ControlPanel`;
 const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 const urlCreateSubscriberList = `${urlDopplerLegacy}/Lists/List`;
-const urlDraft = `${urlDopplerLegacy}/Campaigns/Draft/`;
+const urlDraft = `${urlDopplerLegacy}`;
```
Comment (cbernat) [inline src/i18n/en.js:35]: How about making this change also? to avoid confusion
```suggestion
const homeUrl= `${urlDopplerLegacy}`;
```
You could make it in a separate commit.

File: src/components/DopplerPlus/AgenciesForm.test.js [hunk 2]
```diff
@@ -68,6 +68,11 @@ describe('AgenciesForm component', () => {
     await waitFor(() => {
       return expect(getByText('agencies.success_msg')).toBeInTheDocument();
     });
+
+    expect(screen.getByRole('link', { name: 'agencies.submitted' })).toHaveAttribute(
```
Comment (cbernat) [inline src/components/DopplerPlus/AgenciesForm.test.js:72]: 💯

**Comment by jcamposmk** [comment]
:+1:
